app/views/hub/components/_main_menu.html.erb

```diff
@@ -42,11 +42,13 @@
       </div>
       <% if current_user %>
         <div class="bottom">
-          <%= render "hub/components/menu_item",
-                     link: hub_user_profile_path,
-                     item_label: current_user&.name,
-                     icon_path: image_path("icons/my-profile.svg")
-          %>
+          <% if current_user&.forced_password_reset_at? || current_user&.admin? %>
```

Comment:
Same as above: We now have code that redirects the user to the password change page when needed, so there's no need for this.